File: w7/count_word.py
#!/usr/bin/python
import sys,re
words=[]
count=0
for line in sys.stdin:
        words=re.findall(r'(\b[A-Za-z]+\b)',line)
        for each in words:
		if(each.lower() == sys.argv[1].lower()):
			count+=1;
print sys.argv[1].lower(),"occurred",count,"times";

File: w7/count_word.pl
#!/usr/bin/perl -w
$sum = 0;
while ($line = <STDIN>) {
	@words = split /\W+/, $line;
   $total = @words;
	$num = 0;
	while ($num < $total) {
		if ($words[$num] =~/^$ARGV[0]$/i) {
         $sum += 1;
#         print $words[$num] . "\n";
		}
		$num++;
	}
}
$char = lc($ARGV[0]);
print "$char occurred $sum times\n";
